sregistry/client/__init__.py

```diff
@@ -293,10 +293,9 @@ def help(return_code=0):
     # If the user didn't provide any arguments, show the full help
     if len(sys.argv) == 1:
         help()
-    try:
-        args = parser.parse_args()
-    except:
-        sys.exit(0)
+
```

Comment:
Why did you remove this here?

Reply:
Because if you don't catch the exception, you get a clean exit with value 2, which I think is better than returning with value 0.

Reply:
okay, great.